cherab/core/atomic/line.pyx

```diff
@@ -58,6 +58,16 @@ cdef class Line:
         self.charge = charge
         self.transition = transition
 
+    def __getstate__(self):
+        return {"element": self.element,
+                "charge": self.charge,
+                "transition": self.transition}
+    
+    def __setstate__(self, state):
+        self.element = state["element"]
+        self.charge = state["charge"]
+        self.transition = state["transition"]
```

Comment:
I think the default `__getstate__`,` __setstate__` will do the same. Is there a reason to override them?